Backend/test_project/test_project/settings.py

```diff
@@ -32,10 +32,13 @@
 
     # test_project apps
     'users',
+    "categories.apps.CategoriesConfig",
 
     # third-party apps
     'rest_framework',
     'djoser',
+    'mptt',
+    'django_mptt_admin',
```

Comment:
add this two libraries to requirements.txt (with versions)

Reply:
Done